gravitee-apim-e2e/README.md

```diff
@@ -10,32 +10,32 @@ They are based on Cypress and Jest and can be run against a locally running APIM
 First you need to install the dependencies & build test:
 
 ```shell
-npm install
-npm run build
+yarn install
+yarn build
 ```
 
-Then, the following NPM scripts are available:
- - `npm run test:ui`: Run the APIM stack and the cypress UI tests in docker ([more details](./docker/ui-tests/README.md))
- - `npm run test:ui:dev`: Run the cypress UI tests against a locally running APIM
- - `npm run test:api:mongo`: Run the APIM stack with mongo and the jest API tests in docker ([more details](./docker/api-tests/README.md))
- - `npm run test:api:jdbc`: Run the APIM stack with jdbc and the jest API tests in docker ([more details](./docker/api-tests/README.md))
- - `npm run test:api:dev`: Run the jest API tests against a locally running APIM
- - `npm run apim:serve`: Run the APIM stack in docker with last apim release by default in order to run tests locally from your IDE.
+Then, the following YARN scripts are available:
+ - `yarn test:ui`: Run the APIM stack and the cypress UI tests in docker ([more details](./docker/ui-tests/README.md))
+ - `yarn test:ui:dev`: Run the cypress UI tests against a locally running APIM
+ - `yarn test:api:mongo`: Run the APIM stack with mongo and the jest API tests in docker ([more details](./docker/api-tests/README.md))
+ - `yarn test:api:jdbc`: Run the APIM stack with jdbc and the jest API tests in docker ([more details](./docker/api-tests/README.md))
+ - `yarn test:api:dev`: Run the jest API tests against a locally running APIM
+ - `yarn apim:serve`: Run the APIM stack in docker with last apim release by default in order to run tests locally from your IDE.
     > We have to adapt wiremock according to the gateway location. With this command the gateway is executed in docker, so we have to change the variable to `WIREMOCK_BASE_URL=http://wiremock:8080` in .env
- - `npm run apim:serve -- wiremock`: To run only wiremock on docker  
-   To run with latest master use this env-var `APIM_REGISTRY=graviteeio.azurecr.io APIM_TAG=master-latest npm run apim:serve`
- - `npm run apim:clean`: Stop and remove the containers & volumes.
- - `npm run bulk`: Run jest bulk scripts to create data to local environment
-   - You can pass some options like `npm run bulk -- --apis=50 --applications=5`, available options:
+ - `yarn apim:serve -- wiremock`: To run only wiremock on docker  
+   To run with latest master use this env-var `APIM_REGISTRY=graviteeio.azurecr.io APIM_TAG=master-latest yarn apim:serve`
+ - `yarn apim:clean`: Stop and remove the containers & volumes.
+ - `yarn bulk`: Run jest bulk scripts to create data to local environment
+   - You can pass some options like `yarn bulk -- --apis=50 --applications=5`, available options:
```

Comment:
` yarn bulk --apis=50 --applications=5 `

Reply:
I don't know if there are many, but the ` -- ` throw error with yarn :note-sure: